locksmithing.lic

```diff
@@ -25,56 +28,58 @@ class Locksmithing
     # Practice some music while use lock trainers
     start_script('performance', ['noclean']) unless Script.running?('performance')
     pause 3
+    get_item('lockpick', @lockpick_container) unless @use_lockpick_ring
     daily_lockbox if @daily_trainer
     consumable_lockbox if @consumable_trainers
+    stow_item('lockpick', @lockpick_container) unless @use_lockpick_ring
   end
 
   def daily_lockbox
     if DRSkill.getxp('Locksmithing') < 34
       if @worn_lockbox
-        case bput("remove my #{@box}", 'You take', 'Remove what', "You aren't wearing that", 'You sling','You remove', 'You detach')
+        case DRC.bput("remove my #{@box}", 'You take', 'Remove what', "You aren't wearing that", 'You sling', 'You remove', 'You detach')
         when 'Remove what', "You aren't wearing that"
-          echo('Lockbox is not worn but you declared it is in your yaml, exiting')
+          DRC.message('Lockbox is not worn but you declared it is in your yaml, exiting')
           return
         end
-        bput("close my #{@box}", 'You close', 'That is already',"You can't close that")
+        DRC.bput("close my #{@box}", 'You close', 'That is already', "You can't close that")
       else
-        case bput("get my #{@box}", 'You get a', 'What were')
+        case DRC.bput("get my #{@box}", 'You get a', 'What were')
         when 'What were'
-          echo('Daily Trainer not found! Where is it?')
+          DRC.message('Daily Trainer not found! Where is it?')
           return
         end
       end
 
       pick_box(@box)
 
       if @worn_lockbox
-        bput("pick my #{@box}", 'not making any progress', 'it opens.', "isn't locked", 'The lock feels warm')
-        bput("wear my #{@box}", 'You put', 'You sling','You attach')
+        DRC.bput("pick my #{@box}", 'not making any progress', 'it opens.', "isn't locked", 'The lock feels warm')
```

Comment:
I like the refactoring, but please in the future make it a separate PR. It's hard to see what is prefixing and what is logic changes.

Reply:
Noted, thanks